docs/docs/dcap.md

```diff
@@ -19,19 +9,33 @@ automated for you.
 comes with the necessary libraries bundled, you don't need to do any further setup.
 
+DCAP is also known as ECDSA attestation.
+
 ### DCAP Plugin
-To perform attestation using DCAP, Conclave needs a way to gather information about the platform the enclave is hosted on. This information provides proof from Intel that the system supports SGX and that it is patched and up to date.
+To perform attestation using DCAP, Conclave needs a way to gather information about the platform the enclave is 
+hosted on. This information provides proof from Intel that the system supports SGX and that it is patched and up to 
+date.
+
+DCAP is designed to work on different server topologies. So, rather than directly connecting to Intel services to 
+retrieve this information, the cloud vendor or the owner of the SGX system must provide a DCAP client plugin with 
+the required information. Intel provides a generic DCAP client plugin as part of the DCAP runtime. To use this, you 
+also need to set up a Provisioning Certificate Caching Service (PCCS). Intel provides an example and some 
+instructions [here](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/pccs/README.md). 
+If you would like to use Intel's reference implementation of their [PCCS service](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/pccs), 
+then additional work may be required to provide data in the correct format for Conclave. Please contact R3 support 
+if you need to set this up.
 
-DCAP is designed to work on many different server topologies. So, rather than directly connecting to Intel services to retrieve this information, the cloud vendor or the owner of the SGX system must provide a DCAP client plugin with the required information. Intel provides a generic DCAP client plugin as part of the DCAP runtime. To use this, you also need to set up a Provisioning Certificate Caching Service (PCCS). Intel provides an example and some instructions [here](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/pccs/README.md). If you would like to use Intel's reference implementation of their [PCCS service](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/pccs), then additional work may be required to provide data in the correct format for Conclave. Please contact R3 support if you need to set this up.
+Microsoft has written a DCAP client plugin that works with its Azure Confidential Compute virtual machines. In fact, 
+it also works outside of Azure for single CPU systems, but this may not always be the case.
 
-Microsoft has written a DCAP client plugin that works with its Azure Confidential Compute virtual machines. In fact, it also works outside of Azure for single CPU systems, but this may not always be the case.
+Conclave's bundled Azure client plugin will *only* be used if no other plugin has been found on the system. The 
```

Comment:
This fact should really be moved up where we mention that the Azure plugin is bundled

Reply:
I agree.
In fact the previous sentence `Conclave bundles and uses Azure's DCAP plugin by default` is somehow misleading. If there is another library, the other library might be considered the "default".
So, as Shams suggested, I would refactor such a sentence saying something like
`Conclave bundles the Azure's DCAP plugin, this will be used only if no other plugin has been found on the system.`. Or similar